docs/manual/manual.md

```diff
@@ -755,4 +773,4 @@ For an explanation how to switch between the sounds see
 
 | Document Version | Changes |
 | --- | --- |
-| 0.2 |     |
+| 0.3 | Added howto shutdown  |
```

Comment:
Normally this should be a version table showing all versions, not just the last change...